Formula/voltdb.rb

```diff
@@ -14,13 +14,21 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
 
   def install
     system "ant"
 
-    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader"],
+    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader", "bin/voltdeploy", "bin/voltenv"],
```

Comment:
This way list is a bit easier to maintain. Because `inreplace` gives an error if replace fails and error points to script that needs to be checked on upgrade.